godot-core/src/registry/class.rs

```diff
@@ -232,6 +232,7 @@ fn fill_class_info(item: PluginItem, c: &mut ClassRegistrationInfo) {
             is_editor_plugin,
             is_hidden,
             is_instantiable,
+            ..
```

Comment:
i think it'd be better to explicitly ignore the values we dont use. Since then it's easier to later add new fields as the compiler will tell you where updates are needed. same below.